setup.cfg

```diff
@@ -0,0 +1,236 @@
+    GitHub: issues = https://github.com/deepset-ai/haystack/issues
+    GitHub: repo = https://github.com/deepset-ai/haystack
+description = Neural Question Answering & Semantic Search at Scale. Use modern transformer based models like BERT to find answers in large document collections
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords=
+    QA 
+    Question-Answering 
+    Reader 
+    Retriever 
+    semantic-search 
+    search 
+    BERT 
+    roberta 
+    albert 
+    squad 
+    mrc 
+    transfer-learning 
+    language-model 
+    transformer
+author = deepset.ai
+author_email = malte.pietsch@deepset.ai
+license = Apache License 2.0
+license_file = LICENSE
+platforms = any
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: Freely Distributable
+    License :: OSI Approved :: Apache Software License
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+
+[options]
+use_scm_version = True
+python_requires = >=3.7
+packages = find:
+install_requires =
+    importlib-metadata; python_version < '3.8'
+    torch>1.9,<1.11
+    requests
+    pydantic
+    transformers==4.13.0
+    nltk
+    pandas
+
+    # Utils
+    dill  # pickle extension for (de-)serialization
+    tqdm  # progress bars in model download and training scripts
+    networkx  # graphs library
+    mmh3  # fast hashing function (murmurhash3)
+    quantulum3  # quantity extraction from text
+    azure-ai-formrecognizer==3.2.0b2  # forms reader
+
+    # Preprocessing
+    more_itertools  # for windowing
+    python-docx
+    langdetect   # for PDF conversions
+    tika  # Apache Tika (text & metadata extractor)
+
+    # See haystack/nodes/retriever/_embedding_encoder.py, _SentenceTransformersEmbeddingEncoder
+    sentence-transformers>=0.4.0
+
+    # for stats in run_classifier
+    scipy>=1.3.2  
+    scikit-learn>=1.0.0
+
+    # Metrics and logging
+    seqeval
+    mlflow<=1.13.1 
+
+    # Elasticsearch
+    elasticsearch>=7.7,<=7.10
+    elastic-apm
+
+    # Not found in repo, to review:
+    #httptools
+    #dataclasses-json
+    #uvloop==0.14; sys_platform != 'win32' and sys_platform != 'cygwin'
+
+
+[options.packages.find]
+exclude =
+    rest_api*
+    test*
+    tutorials*
+    ui*
+
+
+[options.extras_require]
+sql = 
+    sqlalchemy>=1.4.2
+    sqlalchemy_utils
+    psycopg2-binary; sys_platform != 'win32' and sys_platform != 'cygwin'
+only-faiss = 
+    faiss-cpu>=1.6.3
+faiss = 
+    farm-haystack[sql,only-faiss]
+only-faiss-gpu = 
+    faiss-gpu>=1.6.3
+faiss-gpu = 
+    farm-haystack[sql,only-faiss-gpu]
+only-milvus = 
```

Comment:
same here, what's the only- packages for? milvus and faiss won't work without sql anyway or am I missing something?

Reply:
See above